### PR TITLE
Provide new `@items` endpoint

### PR DESCRIPTION
**Pull request by vangheem**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
 2.3.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Provide new `@listing` endpoint
```
Comment (cdevienne) [inline CHANGELOG.rst]: The endpoint is actually "@items"

File: guillotina/api/content.py [hunk 2]
```diff
@@ -772,6 +772,88 @@ def getRequestMethod(self):  # noqa
...
+        "default": 20
+    }, {
+        "name": "page",
+        "in": "query",
+        "type": "number",
+        "default": 1
+    }],
+    responses={
+        "200": {
+            "description": "Successfully respect object"
```
Comment (cdevienne) [inline guillotina/api/content.py]: 'respect' does not make sens to me (but english is not my mother tongue)

File: CHANGELOG.rst (hunk 1 above)
Comment (cdevienne) [inline CHANGELOG.rst]: The endpoint is actually "@Items"

File: guillotina/tests/test_api.py [hunk 3]
```diff
@@ -568,3 +568,25 @@ def uninstall(cls, container, request):
...
+        response, status = await requester('GET', '/db/guillotina/@items?page_size=10')
+        assert len(response['items']) == 10
+        assert response['total'] == 22
+
+        response, status = await requester('GET', '/db/guillotina/@items?page_size=10&page=2')
+        assert len(response['items']) == 10
+        assert response['total'] == 22
+
+        response, status = await requester('GET', '/db/guillotina/@items?page_size=10&page=3')
+        assert len(response['items']) == 2
```
Comment (cdevienne) [inline guillotina/tests/test_api.py]: Maybe test include & omit.
Also, on at least one page, check that the returned items are the expected ones.

**Comment by vangheem** [comment]
@cdevienne better now?

**Pull request merged**